src/libraries/Common/src/System/Security/Cryptography/Pkcs/Pkcs12SafeContents.cs

```diff
@@ -287,7 +292,7 @@ public IEnumerable<Pkcs12SafeBag> GetBags()
 
             if (_bags == null)
             {
-                return Enumerable.Empty<Pkcs12SafeBag>();
+                return [];
```

Comment:
`Enumerable.Empty` is in System.Linq. S.S.Cryptography does not have a reference to Linq, and it didn't seem worth it for this one case. An empty collection expression is probably "right" these days anyway.